package.json

```diff
@@ -82,13 +82,15 @@
     ]
   },
   "devDependencies": {
-    "babel-eslint": "7.2.3",
+    "babel-eslint": "^7.2.3",
+    "babel-plugin-inline-import-graphql-ast": "^2.0.0",
```

Comment:
I'm not sure this plugin is reliable or not for now. Theorically, It can be cool if we add it but should we wait for that until v2.0.1 release? What do you think?

also we need to consider these warnings (because we are using next.js):
https://github.com/detrohutt/babel-plugin-inline-import-graphql-ast#known-use-cases
https://github.com/detrohutt/babel-plugin-inline-import-graphql-ast#caveats